lib/theme/Themed.js

```diff
@@ -14,16 +14,24 @@ injectGlobal`
         padding: 0;
         margin: 0;
         background: ${base.colors.background};
+        color: ${base.colors.text};
     }
 `;
 
+const Expand = styled.div`
+  height: 100%;
+  width: 100%;
+  padding: 0;
+  margin: 0;
+`;
+
```

Comment:
Maybe write a little in a README for this module about how to mutate theme.

Reply:
Also for us it would be nice to know what each colors are so we can use them throughout the app consistently

Reply:
At the moment, you can't mutate theme :'( 
You can use the theme in your mutations just fine because it's just [styled-components](https://www.styled-components.com/) theme. 

What do you mean by "know what each colors are so we can use them throughout the app consistently"? There already exists a:

``` js
props.theme.colors.primary
props.theme.colors.danger
props.theme.colors. //ect
```